gsrs-core-entities/src/main/java/ix/core/models/DBGSRSVersion.java

```diff
@@ -0,0 +1,49 @@
+package ix.core.models;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import gov.nih.ncats.common.util.TimeUtil;
+import ix.ginas.models.serialization.GsrsDateDeserializer;
+import ix.ginas.models.serialization.GsrsDateSerializer;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "ix_core_DB_GSRS_version")
```

Comment:
Can we make all lowercase? Shouldn't matter but we always did all lowercase before

Reply:
Yes, they should all be lowercase.